hexa/databases/tests/test_api.py

```diff
@@ -79,20 +77,3 @@ def test_create_database_raise_error(self):
         password = "password"
         with self.assertRaises(ValidationError):
             create_database(bad_input, password)
-
-    def test_create_database_not_access(self):
```

Comment:
Why did this test get deleted?

Reply:
Oh I rename it

Reply:
I'll revert it. I think it was during a rebase with main